.ci.yaml

```diff
@@ -10,6 +10,23 @@ enabled_branches:
   - flutter-\d+\.\d+-candidate\.\d+
 
 platform_properties:
+  staging_build_linux:
```

Comment:
Synced up offline with Yusuf about this. Using staging_build_linux makes more sense the `Linux_staging` as there's quite a bit of infra and dashboarding that would need to change to support the latter. So doing this is quicker.